R/dpinput_sync.R

```diff
@@ -212,19 +207,18 @@ pathnames_reroot <- function(pathnames, new_root = "input_files") {
 }
 
 #' @keywords internal
-sync_iterate <- function(input_map, inputboard_alias, skip_sync, rewrite_ok = F,
+sync_iterate <- function(input_map, board_object, skip_sync, rewrite_ok = F,
                          verbose) {
   synced_map <- purrr::map(.x = input_map, .f = function(input_i) {
-    # This version coincidentally also addresses pins bug where data.txt can be
-    # overwritten
+
     synced_versions <- pins::pin_versions(
```

Comment:
I'm getting a new error here: if the pin doesn't already exist, this throws an error. Need to check if the pin exists first before trying to get versions.